src/commands/mail/address/update.tsx

```diff
@@ -0,0 +1,236 @@
+    To set forwarding addresses, use the --forward-to flag.
+    
+    Use the --catch-all flag to make the mailbox a catch-all mailbox.
+  
+    When running this command with --generated-password the output will be the newly generated and set password.`;
+  static args = { ...mailAddressArgs };
+  static flags = {
+    ...processFlags,
+    ...sharedMailAddressFlags,
+    quota: Flags.integer({
+      description: "mailbox quota in mebibytes",
+    }),
+    password: Flags.string({
+      summary: "mailbox password",
+      description:
+        "If set, the mailbox will be updated to this password. If omitted, the password will remain unchanged.\n\nCAUTION: providing this flag may log your password in your shell history!",
+    }),
+    "random-password": Flags.boolean({
+      summary: "generate a random password",
+      description:
+        "This flag will cause the command to generate a random 32-character password for the mailbox; when running with --quiet, the password will be printed to stdout.",
+    }),
+    "forward-to": Flags.string({
+      summary: "forward mail to another address",
+      multiple: true,
+      description:
+        "This flag will cause the mailbox to forward all incoming mail to the given address.\n\nNote: This flag is exclusive with --catch-all, --quota, --password and --random-password.",
+      relationships: [
+        {
+          type: "none",
+          flags: ["catch-all", "quota", "password", "random-password"],
+        },
+      ],
+    }),
+  };
+
+  static examples = [
+    {
+      description: "Update non-interactively with password",
+      command:
+        "$ read -s PASSWORD &&\n <%= config.bin %> <%= command.id %> --password $PASSWORD --address foo@bar.example",
+    },
+    {
+      description: "Update non-interactively with random password",
+      command:
+        "<%= config.bin %> <%= command.id %> --random-password --address foo@bar.example",
+    },
+    {
+      description: "Update forwarding address",
+      command:
+        "<%= config.bin %> <%= command.id %> --address foo@bar.example --forward-to bar@bar.example --forward-to baz@bar.example",
+    },
+  ];
+
+  protected async setAddress(
+    mailAddressId: string,
+    address: string,
+    process: ProcessRenderer,
+  ): Promise<void> {
+    await process.runStep("set the address of a mail address", async () => {
+      const response = await this.apiClient.mail.updateMailAddressAddress({
+        mailAddressId: mailAddressId,
+        data: {
+          address: address,
+        },
+      });
+
+      assertStatus(response, 204);
+    });
+  }
+
+  protected async setCatchAll(
+    mailAddressId: string,
+    catchAll: boolean,
+    process: ProcessRenderer,
+  ): Promise<void> {
+    await process.runStep("set the catchall of a mail address", async () => {
+      const response = await this.apiClient.mail.updateMailAddressCatchAll({
+        mailAddressId: mailAddressId,
+        data: {
+          active: catchAll,
```

Comment:
`active`? Wrong field?

Reply:
Nope, thats just how bool is called in the openApi 🤷‍♂️